### PR TITLE
feat: Promotion

### PR DESCRIPTION
**Pull request by slzhffktm**
Resolves https://github.com/xendit/xendit-go/issues/34

### CODE REVIEW

File: card/client.go [hunk 1]
```diff
@@ -74,8 +74,8 @@ func (c *Client) CaptureChargeWithContext(ctx context.Context, data *CaptureChar
 }
 
 // GetCharge gets a card charge
-func (c *Client) GetCharge(data *CaptureChargeParams) (*xendit.CardCharge, *xendit.Error) {
```
Comment (bxcodec) [inline card/client.go:77]: Isn't this going to be a breaking change?

File: card/client.go [hunk 2]
```diff
@@ -74,8 +74,8 @@ func (c *Client) CaptureChargeWithContext(ctx context.Context, data *CaptureChar
...
-	return c.CaptureChargeWithContext(context.Background(), data)
+func (c *Client) GetCharge(data *GetChargeParams) (*xendit.CardCharge, *xendit.Error) {
```
Comment (mychaelgo) [inline card/client.go:77]: why this changed?

File: card/client.go (hunk 1 above)
Comment (bxcodec) [inline card/client.go:77]: Isn't this going to be a breaking change?

Comment (bxcodec) [reply card/client.go:77]: Can we keep it to be a minor release?
I don't see any needs for major releae

Comment (stanleynguyen) [reply card/client.go:77]: I think this was a bug from previously

File: card/client.go (hunk 2 above)
Comment (stanleynguyen) [reply card/client.go:77]: I think this was a bug from previously

File: card/client.go (hunk 1 above)
Comment (bxcodec) [reply card/client.go:77]: So are we going to release it as a major version then?

File: card/client.go (hunk 2 above)
Comment (slzhffktm) [reply card/client.go:77]: yeah sorry, should I separate this change into another PR?

File: card/client.go (hunk 1 above)
Comment (stanleynguyen) [reply card/client.go:77]: Hmm I think just keep it in this minor. I think we haven't gotten complaints cos no one is using this feature yet

File: card/client.go (hunk 2 above)
Comment (stanleynguyen) [reply card/client.go:77]: No I think it's fine being kept here as it's a bug fix rather (eventhough it broke the signature)

Comment (mychaelgo) [reply card/client.go:77]: ah okay.

**Review by mychaelgo — APPROVED** [review]
LGTM 👍

**Review by bxcodec — APPROVED** [review]
LGTM. 

I'm worried about the versioning later.

**Pull request merged**